### DISCUSSION TITLE
绳网精选-1.1版本音擎上新 | 第一弹

### DISCUSSION THREAD

**Discussion by winiking** [share121/inter-knot#39] [其他]
![zzz](https://github.com/user-attachments/assets/1acf019b-0be8-4473-a678-f82525089932)

绳网精选，绳匠之友！
1.1版本最新款音擎——「玉壶青冰」&「鎏金花信」强势来袭！
带上我们的定制音擎，让您每一次的空洞探索都如履平地！

—— 欢迎来到新艾利都！——
▼ 《绝区零》1.1版本「卧底蓝调」将于2024年8月14日正式上线！
bilibili安卓端下载地址：https://www.biligame.com/detail/?id=108434
bilibiliPC端下载地址：https://www.biligame.com/detail/?id=111210
![玉壶青冰](https://github.com/user-attachments/assets/6604dd73-554b-4f79-aca1-dca4fcab775e)
![玉壶青冰](https://github.com/user-attachments/assets/9ec90105-1501-44e1-a77c-0d4d83e93290)
![鎏金花信](https://github.com/user-attachments/assets/c8c06462-d4b7-46b9-ac63-8493d867499b)
![鎏金花信](https://github.com/user-attachments/assets/fa77162f-8af0-4dbf-a825-26c1683e8521)

**Comment by yyxzz808** [comment]
好耶

**Comment by ImHaoYuan** [comment]
太棒了
